### ISSUE TITLE
[Flag] - There are 2 hidden flags

### ISSUE THREAD

**Issue by rubenpro** [Semantic-Org/Semantic-UI#6926]
**Steps to Reproduce**
1. Go to https://semantic-ui.com/elements/flag.html
2. Go to Types and see the full flag list.
3. If you count, you get 245 flags.

**Expected**
We must have the same exact number of flags in the flags.png sprite, which is used for the component (https://github.com/Semantic-Org/Semantic-UI/blob/master/dist/themes/default/assets/images/flags.png)

**Result**
Instead, we got 2 more. One of these are the Catalan (Catalonia), which you can see appearing under the Canadian flag (1st col, 36th row). I can't spot the last one.

**Comment by TheJltres** [comment]
You can get Catalan flag by adding

`i.flag.es.ca:before, i.flag.catalonia:before {
    background-position: 0 -936px
}`

**Comment by rubenpro** [comment]
It's what I already did but you get this output anyway:

![Selection_003](https://user-images.githubusercontent.com/36886621/86218579-4ae63e80-bb81-11ea-82f7-bf70bf8cbc04.png)

I think they should improve it :/

**Comment by TheJltres** [comment]
Here you have an example: https://jsfiddle.net/31d6y7mn

**Comment by rubenpro** [comment]
Yeah of course, the Flag component, internally uses the "i" html tag, which uses those classes, but if you change one of those flags with the names you say "es ca" or "catalonia", doesn't work. You can try it in there:

https://codesandbox.io/s/4t34j?module=/example.js&file=/example.js:160-166

**Comment by TheJltres** [comment]
https://codesandbox.io/s/semantic-ui-example-g5wwo

I imported App.css on index.js, and added a flag with the other one.

**Comment by TheJltres** [comment]
I did a PR in Fomantic-UI, the active fork of Semnatic-UI

https://github.com/fomantic/Fomantic-UI/pull/1549

**Comment by rubenpro** [comment]
Yeah thanks, I hope they fix it soon because as you can see in the codesandbox link you provided before, console output show us the warning.

**Comment by TheJltres** [comment]
Yes, the problem is with types of React-Semantic, now with Fomatic-UI v3 (https://github.com/fomantic/Fomantic-UI/blob/master/ROADMAP.md) will have React types also!

**Comment by lubber-de** [comment]
> Yeah thanks, I hope they fix it soon because as you can see in the codesandbox link you provided before, console output show us the warning.

We most probably won't merge the above mentioned PR. (Technical reasons described at https://github.com/fomantic/Fomantic-UI/pull/1549#issuecomment-652402724)
But even, if we would , as you are referring to an error in SUI-React, it would have to be patched there, otherwise you will still encounter the error message you are referring to.
If you want this fixed, you have to raise an issue at https://github.com/Semantic-Org/Semantic-UI-React

**Comment by rubenpro** [comment]
Thank you, I've posted it in [there](https://github.com/Semantic-Org/Semantic-UI-React/issues/4077).
